benchmark/solver/solver.cpp

```diff
@@ -43,12 +43,14 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <iostream>
 
 
+#include <ginkgo/core/preconditioner/isai.hpp>
+
+
```

Comment:
Shouldn't this already be included in Line 33 with `<ginkgo/ginkgo.hpp>`, as it is a public Ginkgo header?

Reply:
should in the worst case also be imported in preconditioners.hpp